linkerd/app/src/oc_collector.rs

```diff
@@ -17,6 +17,7 @@ pub enum Config {
     Disabled,
     Enabled {
         control: ControlConfig,
+        labels: Option<HashMap<String, String>>,
```

Comment:
nit, take it or leave it: Since we use `Option::unwrap_or_default` later, if this is `None`, we will eventually create an empty `HashMap`. Seems like we could just pass an empty hash map if there were no labels. Since [`HashMap::new` doesn't allocate](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.new), there isn't a lot of additional overhead (it's a _little_ bigger than an `Option` but this isn't a hot path).